assets/large_language_models/rag/components/src/data/assetid

```diff
@@ -0,0 +1 @@
+azureml://subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourcegroups/baker-static/providers/Microsoft.MachineLearningServices/workspaces/baker-eastus2/data/tidy-ring-78lzkwl4fn/versions/1
```

Comment:
What's this?

Reply:
This asset is not public right?

Reply:
oop this was added by accident from my testing, let me remove